common/flatpak-dir.c

```diff
@@ -7139,6 +7139,8 @@ export_desktop_file (const char         *app,
                 g_string_append_printf (new_exec, " @@ %s @@", arg);
               else if (strcasecmp (arg, "%u") == 0)
                 g_string_append_printf (new_exec, " @@u %s @@", arg);
+              else if (strcmp (arg, "@@") == 0 || strcmp (arg, "@@u") == 0)
```

Comment:
Since we already have two special tokens, `@@` and `@@u`, I'd be tempted to assume we will eventually need to invent a third, and assume that it will be `@@x` or something:

```suggestion
              else if (g_str_has_prefix (arg, "@@"))
```

Reply:
yeah, seems best to just reserve the entire @@ prefix

Reply:
#4156 incorporates this suggestion.